src/main/java/org/interledger/codecs/oer/OerUint256Codec.java

```diff
@@ -0,0 +1,119 @@
+package org.interledger.codecs.oer;
+
+import org.interledger.codecs.Codec;
+import org.interledger.codecs.CodecContext;
+import org.interledger.codecs.oer.OerUint256Codec.OerUint256;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * <p>
+ * An extension of {@link Codec} for reading and writing an ASN.1 OER 256-Bit integer type as
+ * defined by the Interledger ASN.1 definitions.
+ * </p>
+ * <p>
+ * All Interledger ASN.1 integer types are encoded as fixed-size, non-extensible numbers. Thus, for
+ * a UInt256 type, the integer value is encoded as an unsigned binary integer in 32 octets.
+ * </p>
+ */
+public class OerUint256Codec implements Codec<OerUint256> {
+
+  /**
+   * ASN.1 256BitUInt: Alias for a fixed length octet string of 32 octets.
+   *
+   * @param context An instance of {@link CodecContext}.
+   * @param inputStream An instance of @link InputStream}.
+   * @throws IOException If there is a problem writing to the {@code stream}.
+   * @throws IllegalArgumentException If the input has a value greater than 18446744073709551615.
+   */
+  @Override
+  public OerUint256 read(final CodecContext context, final InputStream inputStream)
+      throws IOException {
+    Objects.requireNonNull(context);
+    Objects.requireNonNull(inputStream);
+
+    final byte[] returnable = new byte[32];
+    inputStream.read(returnable);
```

Comment:
inputstream.read(byte[]) does not guarantee that it will fill the array, so we should check that it read the required number.